sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateQcTypeDaoTest.java

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+package uk.ac.bbsrc.tgac.miso.persistence.impl;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.hibernate.SessionFactory;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import uk.ac.bbsrc.tgac.miso.AbstractDAOTest;
+import uk.ac.bbsrc.tgac.miso.core.data.type.QcType;
+
+public class HibernateQcTypeDaoTest extends AbstractDAOTest {
+
+  @Rule
+  public final ExpectedException exception = ExpectedException.none();
+
+  @Autowired
+  private SessionFactory sessionFactory;
+
+  @InjectMocks
+  private HibernateQcTypeDao dao;
+
+  @Before
+  public void setup() throws IOException {
+    MockitoAnnotations.initMocks(this);
+    dao.setSessionFactory(sessionFactory);
+  }
+
+  @Test
+  public void testListAllSampleQcTypes() throws IOException {
+    Collection<QcType> qcTypes = dao.list();
```

Comment:
Doesn't do what method name suggests - is not limited to Sample QCs. Rename (and others in this class)